src/adapters/supabase/helpers/logs.ts

```diff
@@ -0,0 +1,391 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
If I remember correctly you had fixed the `any` in some other pull request, please do so here as well